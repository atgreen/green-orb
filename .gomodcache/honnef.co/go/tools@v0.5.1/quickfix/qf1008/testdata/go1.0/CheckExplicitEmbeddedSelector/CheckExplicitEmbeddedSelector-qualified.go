package pkg

import (
	"io"

	assist "example.com/CheckExplicitEmbeddedSelectorassist"
)

func fnQualified() {
	_ = io.EOF.Error  // minimal form
	_ = assist.V.T2.F //@ diag(`could remove embedded field "T2" from selector`)
	_ = assist.V.F    // minimal form
}
