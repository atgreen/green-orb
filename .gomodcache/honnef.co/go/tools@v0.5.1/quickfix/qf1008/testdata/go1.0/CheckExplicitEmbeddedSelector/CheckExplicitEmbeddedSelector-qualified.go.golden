package pkg

import (
	"io"

	assist "example.com/CheckExplicitEmbeddedSelectorassist"
)

func fnQualified() {
	_ = io.EOF.Error // minimal form
	_ = assist.V.F   //@ diag(`could remove embedded field "T2" from selector`)
	_ = assist.V.F   // minimal form
}
